holmes.py

```diff
@@ -250,6 +262,9 @@ def alertmanager(
         elif destination == DestinationType.SLACK:
             slack.send_issue(issue, result)
 
+    if dump_raw_output:
```

Comment:
Can you create a shared function to dump the json that both commands use? (And please add it to the other commands as well for consistency.)

I suspect there is an idiomatic way to dump pydantic models to file that will work and be re-usable in all the calls.

Reply:
done